packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -79,11 +79,19 @@ export const startPSM = async (
       E(E(zoe).getInvitationIssuer()).getAmountOf(poserInvitationP),
     ]);
 
+  const [anchorInfo, stableInfo] = await Promise.all(
+    [anchorBrand, stable].map(b => E(b).getDisplayInfo()),
```

Comment:
consider a note about why "display" info is reliable for this use case

Reply:
I don't have an answer for that. Can you write something that would justify it?